standard-integration/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PayPal JS SDK Standard Integration</title>
+  </head>
+  <body>
+    <div id="paypal-button-container"></div>
+    <!-- Replace the "test" client-id value with your client-id -->
+    <script src="https://www.paypal.com/sdk/js?client-id=test&currency=USD"></script>
+    <script>
+      paypal
+        .Buttons({
+          createOrder: async (data) => {
+            try {
+              const response = await fetch('/api/orders', {
+                method: 'POST',
+                headers: {
+                  'Content-Type': 'application/json',
+                },
+                // use the "body" param to optionally pass additional order information
+                // like product ids and quantities
+                body: JSON.stringify({
+                  cart: [
+                    {
+                      id: 'YOUR_PRODUCT_ID',
+                      quantity: 'YOUR_PRODUCT_QUANTITY',
+                    },
```

Comment:
option to include `purchase_units` as an example here

Reply:
Unfortunately we cannot recommend having purchase_units in the frontend payload, at least not in the Orders API format. The amount has to be managed on the server-side to prevent any malicious actors from tampering with it via Chrome Dev Tools.